<launch>
  <rosparam command="load" file="$(find saisun3d)/config.yaml" />
  <node pkg="saisun3d" type="saisun.py" name="" output="screen">
    <remap from="~config" to="/config/saisun" />
    <remap from="~param" to="/saisun" />
    <remap from="~scene" to="/sensors/capt_pc2" />
  </node>
</launch>
